data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+			`	The man starts talking into some communication device, giving orders to the fleet that will be joining you on the mission, then puts the device back on his wasit and looks to you. "Also, since we'll be going into battle together, I might as well introduce myself. I'm Deep Lieutenant Shane Paris. Good luck fighting."`
+				accept
+	
+	npc save escort accompany
+		government Republic
+		personality heroic escort
+		ship "Corvette (Speedy)" "D.S.S. Faraday"
+	
+	on visit
+		dialog `You have landed on <planet>, but either you left the <npc> behind or you didn't pick up the rest of your escorts! Wait for the <npc> to enter the system before landing or make sure you have picked up the escorts on Memory and Farpoint.`
+
+mission "Bactrian Unlock [1]: Pirates"
+	invisible
+	landing
+	to offer
+		has "Bactrian Unlock [0]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Firebird (Plasma)" "The Court Jester"
+		ship "Firebird (Plasma)" "The Knighted One"
+		ship "Headhunter (Particle)" "Cerberus"
+		ship "Headhunter (Particle)" "Charon"
+
+mission "Bactrian Unlock [1]: Escorts"
+	invisible
+	landing
+	source Memory
+	to offer
+		has "Bactrian Unlock [1]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: offered"
+			has "Bactrian Unlock [3-alt]: offered"
+	
+	npc escort kill
+		government Republic
+		personality heroic escort
+		fleet
+			names "deep"
+			variant
+				"Mule (Heavy)" 2
+				"Aerie"
+				"Dagger" 2
+	
+	on offer
+		conversation
+			`You notice a group of Deep Security warships, two Mules and an Aerie, loading up on ammunition just outside of the spaceport. After thirty minutes they tell you that they are ready to launch and join in on the rescure mission.`
+				accept
+
+mission "Bactrian Unlock [1]: Reinforcements"
+	invisible
+	landing
+	source Farpoint
+	to offer
+		has "Bactrian Unlock [1]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: offered"
+			has "Bactrian Unlock [3-alt]: offered"
+	
+	npc escort kill
+		government Republic
+		personality heroic escort
+		fleet
+			names "republic capital"
+			fighters "republic fighter"
+			variant
+				"Cruiser" 2
+				"Combat Drone" 8
+	
+	on offer
+		conversation
+			`While landing on <planet> to refuel, you are approached by two Navy officers who ask if you are involved in the rescue mission. When you respond yes, they inform you that they will be bringing two Cruisers to assist in retrieving the Star Queen.`
+				accept
+
+mission "Bactrian Unlock [2]"
+	landing
+	name "Return to <destination>"
+	description "Now that you have secured the Star Queen, return to <destination> with it intact."
```

Comment:
"with it intact" is OK. Some alternatives could be

- "...,  escort it safely to \<destination\>."
- "..., return it to \<destination\> intact."
- "..., ensure its survival en route to \<destination\>"